adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -34,24 +34,43 @@
 from adafruit_esp32spi import adafruit_esp32spi
 import adafruit_esp32spi.adafruit_esp32spi_requests as requests
 
+class WiFiConnType: # pylint: disable=too-few-public-methods
+    """An enum-like class representing the different types of WiFi connections
+    that can be made. The values can be referenced like ``WiFiConnType.normal``.
+    Possible values are
+    - ``ThermocoupleType.normal``
+    - ``ThermocoupleType.enterprise``
+    """
+    # pylint: disable=invalid-name
+    normal = 1
+    enterprise = 2
+
 class ESPSPI_WiFiManager:
     """
     A class to help manage the Wifi connection
     """
-    def __init__(self, esp, secrets, status_pixel=None, attempts=2):
+    def __init__(self, esp, secrets, status_pixel=None, attempts=2, wificonntype=WiFiConnType.normal):
         """
         :param ESP_SPIcontrol esp: The ESP object we are using
         :param dict secrets: The WiFi and Adafruit IO secrets dict (See examples)
         :param status_pixel: (Optional) The pixel device - A NeoPixel or DotStar (default=None)
         :type status_pixel: NeoPixel or DotStar
         :param int attempts: (Optional) Failed attempts before resetting the ESP32 (default=2)
+        :param const con_type: (Optional) Type of WiFi connection: normal=1, WPA2 Enterprise=2
+        :param ~adafruit_esp32spi_wifimanager.WiFiConnType wificonntype: The type of WiFi \
+        connection to make. The default is "normal".
         """
         # Read the settings
         self._esp = esp
         self.debug = False
         self.ssid = secrets['ssid']
         self.password = secrets['password']
+        self.ent_ssid = secrets['ent_ssid']
```

Comment:
I feel using the regular `secrets['ssid']` is OK here.

Reply:
I thought about that, but I like having one secrets.py file with all of the options in it. At the very least, it makes testing easier, since I'm switching back and forth testing both regular and enterprise connections during development. I'm not going to get to it tonight, but what do you think about adding some code that would support both, but fall back to the regulat ssid if the ent_ssid wasn't in the secrets.py file?

Reply:
Detecting the `ent_ssid` and falling back to `ssid` would be a great solution. My main issue with the way that this was done was the creation of more variables (possibly unused/broken, wouldnt find `secrets['ssid']` at all).

Reply:
OK...I took a stab at this in my latest code. Commit coming soon!